app/views/posts/index.html.erb

```diff
@@ -1 +1,3 @@
-<h1>These are the posts for the user</h1>
\ No newline at end of file
+<%= render @user%>
+
+<%= render partial: "posts/postandcomments", collection:@posts %>
```

Comment:
Same here. The post should also display the **Post Number** as you can see from the wireframe. Kindly add the same. ⬇️

| Your design  | Wireframe design |
| ------------- | ------------- |
| ![image](https://github.com/MosDevx/BlogApp/assets/33268927/b0ce2608-2a3c-4f1f-8e0a-4e52dd5a0843) | ![image](https://github.com/MosDevx/BlogApp/assets/33268927/97e51987-8106-420d-949f-4c11b5a91f44) |

Reply:
Implemented